### DISCUSSION TITLE
Parsing error: Cannot find module 'next/babel'

### DISCUSSION THREAD

**Discussion by EugeneGohh** [vercel/next.js#26800] [Help] [answered]
I am having this issue after I setup my using `npx create-next-app`. The problem is in the picture as shown below:

![Untitled](https://user-images.githubusercontent.com/64187129/124082970-eacdf500-da7f-11eb-9797-fad1a468a15d.png)

May I know how can I resolve this? This is my first time building a project with Next.js.

**Comment by EugeneGohh** [comment, marked as answer]
Never mind, issue solved.

**Reply by jayhubb4** [reply]
What did you have to do? How did you solve this? I'm having an issue deploying and I have a similar issue.
